### DISCUSSION TITLE
Updating RENOVATE_GIT_AUTHOR and updating last commits

### DISCUSSION THREAD

**Discussion by Charlie-Crowdstrike** [renovatebot/renovate#29308] [Request Help] [answered]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

Bitbucket

### Please tell us more about your question or problem

When updating `RENOVATE_GIT_AUTHOR` to a different email, all future builds for branches/PR's fail due to 

> Last commit author does not match git author email - branch has been modified

This is understandable and documented behaviour, as we do not want renovate overwriting manual changes made by other users, but we are wondering the best way to propagate a change to the renovate bot's email itself.

One option we considered was manually rebasing and amending the last commit author for all open PR's/branches, but is there a convention in place for this use case?

Another option we considered was adding a script in the pipeline to check the last author for each branch renovate crawls, and if it matches the old renovate author email address we updated from, we would run a 
`git commit --amend --author="New Renovate Author <email@address.com>" --no-edit` before other actions.

Any guidance here would be helpful, thank you!

### Logs (if relevant)

<details><summary>Logs</summary>

```

DEBUG: Last commit author does not match git author email - branch has been modified

```

</details>

**Comment by rarkins** [comment, marked as answer]
gitIgnoredAuthors
